### ISSUE TITLE
Add AlphaWallet to apkure

### ISSUE THREAD

**Issue by colourfreak** [AlphaWallet/alpha-wallet-android#2203]
https://apkpure.com/

**Comment by JamesSmartCell** [comment]
@colourfreak do you have any info about this site? I tried to upload AW Android on to it, it uploaded, but when I added it to the developer console it said 'This application has existed on Google Play". I have contacted their developer support, see what they say.

**Comment by colourfreak** [comment]
<img width="1049" alt="Screenshot 2021-11-15 at 08 25 38" src="https://user-images.githubusercontent.com/51817359/141746888-4bc25510-5e70-4864-9f39-c3b0a657ec69.png">

**Comment by hboon** [comment]
Does that mean someone else uploaded this in our name? Can we take over it?

**Comment by JamesSmartCell** [comment]
> Does that mean someone else uploaded this in our name? Can we take over it?

I don't think it works like that ... there is no 'ownership' it's kind of weird. I was able to upload the latest APK just by supplying the binary. However I don't know how we can update the page, their support was very fast to contact me back though. I'll ask.

**Comment by colourfreak** [comment]
@hboon Shall we transfer this issue to AlphaWallet-web3-provider repo?

**Comment by hboon** [comment]
@colourfreak no, it's good here. https://github.com/AlphaWallet/AlphaWallet-web3-provider is for the code that we inject into the dapp browser to support dapps.

**Comment by colourfreak** [comment]
@seabornlee @justindg Can you take care of this issue?

**Comment by seabornlee** [comment]
@colourfreak Ok, I will take care of this.

**Comment by seabornlee** [comment]
<img width="772" alt="image" src="https://user-images.githubusercontent.com/735708/159209180-7aea35df-4f6a-4357-b457-8787a02a84ef.png">

The latest version is uploaded, I have no idea who did it. Maybe their operation team?

**Comment by colourfreak** [comment]
> <img alt="image" width="772" src="https://user-images.githubusercontent.com/735708/159209180-7aea35df-4f6a-4357-b457-8787a02a84ef.png">
> 
> The latest version is uploaded, I have no idea who did it. Maybe their operation team?

Can you please reupload with updated [graphics](https://github.com/AlphaWallet/alpha-wallet-android/issues/2446) + [icon](https://github.com/AlphaWallet/alpha-wallet-android/issues/2447)?

**Comment by justindg** [comment]
> > <img alt="image" width="772" src="https://user-images.githubusercontent.com/735708/159209180-7aea35df-4f6a-4357-b457-8787a02a84ef.png">
> > The latest version is uploaded, I have no idea who did it. Maybe their operation team?
> 
> Can you please reupload with updated [graphics](https://github.com/AlphaWallet/alpha-wallet-android/issues/2446) + [icon](https://github.com/AlphaWallet/alpha-wallet-android/issues/2447)?

Haven't applied the new playstore graphics (I will work on it), the new icons however are already updated, but #2443 needs to be merged first :)

**Comment by hboon** [comment]
Out of phase 1
